cli/data.go

```diff
@@ -24,7 +24,7 @@ import (
 const (
 	dataDir                  = "data"
 	metadataDir              = "metadata"
-	defaultParallelDownloads = 10
+	defaultParallelDownloads = 100
```

Comment:
Thanks for updating here. You had tested that there aren't significant speedups beyond 100 parallel downloads, correct?

Reply:
Just tested exporting 2000 files with 100 vs 1000, and they were within ~100ms of each other (~2.9s vs ~3s)